snaps/reference/permissions.md

```diff
@@ -3,22 +3,23 @@ description: See the Snaps permissions reference.
 sidebar_position: 4
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Snaps permissions
 
-Your Snap can [request the following permissions](../how-to/request-permissions.md).
+Snaps can [request the following permissions](../how-to/request-permissions.md).
 
 ## RPC API permissions
 
-You must request permission to use any
-[restricted JSON-RPC API methods](rpc-api.md#restricted-methods).
-
-For example, to request to use [`snap_dialog`](rpc-api.md#snap_dialog), add the following to the
+You must request permission to call [Snaps API methods](snaps-api.md).
+For example, to request to call [`snap_dialog`](snaps-api.md#snap_dialog), add the following to the
```

Comment:
```suggestion
For example, to call the [`snap_dialog`](snaps-api.md#snap_dialog) function, add the following to the
```

Reply:
If you read the sentence before this, it really is "requesting to call" an API method. And to be consistent across the docs, we should refer to API methods as "methods" rather than "functions." In most of our docs, it's also okay to refer directly to the method names without saying "the X method" (in this case, we mention "methods" directly in the previous sentence), so I'll leave this as is.